bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -621,7 +621,7 @@
 		<item-type>Number:Time</item-type>
 		<label>Operating Hours</label>
 		<description>The operating hours</description>
-		<state readOnly="true" pattern="%d %unit%"/>
```

Comment:
Even though the default unit is hours, perhaps the label should say "operating time" rather than "operating hours" just in case user applies a different UOM ?

Reply:
I can change label and description, but is it really worth it given the channel is also named `operatingHours`? I mean, after all, it's not really wrong.

Reply:
Yeah. OK.